js/apps/system/_admin/aardvark/APP/react/build/asset-manifest.json

```diff
@@ -1,26 +1,26 @@
 {
-  "main.css": "static/css/main.7eea4f36.chunk.css",
-  "main.js": "static/js/main.5acb84ce.chunk.js",
-  "main.js.map": "static/js/main.5acb84ce.chunk.js.map",
+  "main.css": "static/css/main.6e703ac5.chunk.css",
+  "main.js": "static/js/main.7c5e95b6.chunk.js",
+  "main.js.map": "static/js/main.7c5e95b6.chunk.js.map",
   "static/css/1.0f119f1e.chunk.css": "static/css/1.0f119f1e.chunk.css",
-  "static/js/1.eb9b5434.chunk.js": "static/js/1.eb9b5434.chunk.js",
-  "static/js/1.eb9b5434.chunk.js.map": "static/js/1.eb9b5434.chunk.js.map",
+  "static/js/1.5455700b.chunk.js": "static/js/1.5455700b.chunk.js",
```

Comment:
Those changes are unrelated?

Reply:
I believe the name is expected to change with every build (at least if something was modified), not sure if this done to avoid caching issues when served.